WordPress/src/main/java/org/wordpress/android/ui/posts/AddCategoryFragment.java

```diff
@@ -136,4 +130,10 @@ public void onSaveInstanceState(Bundle outState) {
         super.onSaveInstanceState(outState);
         outState.putSerializable(WordPress.SITE, mSite);
     }
+
+    @Override
+    public void onDestroy() {
+        super.onDestroy();
+        mBinding = null; // ensures binding reference gets null on fragment destroy
```

Comment:
Minor (🔍): This comment is not too necessary as it is self-explanatory what is happening here and why. No need for you to do anything at this point, this is just an FYI for you for your next PRs.

Reply:
Updated the code.

Reply:
Great, thanks for that @neeldoshii , although, as I told you this was totally optional and you could have avoided it, I added this comment for educational purposes only! 🥇 

FYI: However, I want to emphasise here again on avoiding to rebase + force push on any PR ([context](https://github.com/wordpress-mobile/WordPress-Android/pull/20941#discussion_r1636312985)).

Reply:
Sure 👍.